Drv/TcpClient/test/ut/TcpClientTester.cpp

```diff
@@ -36,6 +36,11 @@ void TcpClientTester ::test_with_loop(U32 iterations, bool recv_thread) {
     server.configure("127.0.0.1", port, 0, 100);
     this->component.configure("127.0.0.1", port, 0, 100);
     serverStat = server.startup();
+    if (serverStat != SOCK_SUCCESS)
+    {
+        printf("TCP server startup error: %s\n", strerror(errno));
+        printf("Port: %u\n", port);
+    }
```

Comment:
As a note, this could be done with the following on line 44:

```
ASSERT_EQ(serverStat, SOCK_SUCCESS) <<  "TCP server startup error:" << strerror(errno) << " Port:" << port;
```

Reply:
I agree, I made the changes.

I can see that it is currently done both ways in the unit tests. I might try to uniform that in the future.